### PR TITLE
[ADVAPP-595] Restrict viewing of super admin users to other super admin users

### PR DESCRIPTION
**Pull request by ankit-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/ADVAPP-595

### Technical Description

> Describe your changes in detail. This should include technical/architectural decisions and reasoning, if applicable.
> Only Super Admin Role user can view other "Super Admin" users listed in /user-management. 
_______________________________________________

### CODE REVIEW

**Review by Orrison — CHANGES_REQUESTED** [review]
@ankit-canyon this other PR has been merged in, you should be clear to work to resolve this merge conflict now.

**Review by mxm1070 — DISMISSED** [review]

File: app/Filament/Resources/UserResource.php [hunk 1]
```diff
@@ -78,80 +63,14 @@ class UserResource extends Resource
...
-            ])
-            ->bulkActions([
-                BulkActionGroup::make([
-                    DeleteBulkAction::make(),
-                    AssignLicensesBulkAction::make()
-                        ->visible(fn () => auth()->user()->can('create', License::class)),
-                ]),
-            ])->defaultSort('name', 'asc');
+        return parent::getEloquentQuery()
+            ->when(! auth()->user()->getIsAdminAttribute(), function (Builder $query) {
```
[the remark this answers is not in the document]
Comment (Orrison) [reply app/Filament/Resources/UserResource.php:69]: https://juststeveking.uk/articles/tappable-query-scopes-in-laravel/

[the remark this answers is not in the document]
Comment (Orrison) [reply app/Filament/Resources/UserResource.php:69]: We probably don't need the attribute if we are not using it anywhere. And just create a Tappable Scope to use.

**Comment by ankit-canyon** [comment]
HI
@mxm1070 @Orrison 

All suggestions has been done and implemented. 

Thank you.

**Review by Orrison — APPROVED** [review]

**Pull request merged**